datalake/src/main/java/com/sequenceiq/datalake/flow/detach/event/DatalakeResizeRecoveryFlowChainStartEvent.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.datalake.flow.detach.event;
+
+import java.util.Objects;
+
+import com.sequenceiq.datalake.entity.SdxCluster;
+import com.sequenceiq.datalake.flow.SdxEvent;
+
+public class DatalakeResizeRecoveryFlowChainStartEvent extends SdxEvent {
```

Comment:
could you add `toString` implementation?

Reply:
Done.